src/deferredtask.cc

```diff
@@ -2,26 +2,28 @@
 
 namespace exawallet {
 
-DeferredTask::DeferredTask(v8::Isolate* isolate) {
-    resolver.Reset(isolate, v8::Promise::Resolver::New(isolate->GetCurrentContext()).ToLocalChecked());
+DeferredTask::DeferredTask() {
+    v8::Local<v8::Promise::Resolver> value =v8::Promise::Resolver::New(Nan::GetCurrentContext()).ToLocalChecked();
```

Comment:
why did you rewrite it?

Reply:
There were some compilation errors I managed to fix introducing explicit type for value variable.